blob/service.go

```diff
@@ -121,6 +121,7 @@ func (s *Service) GetAll(ctx context.Context, height uint64, namespaces []share.
 		resultErr   = make([]error, len(namespaces))
 	)
 
+	log.Debugw("getting blobs for", "height", height, "namespaces", namespaces)
```

Comment:
nit: i'd probably just have "getting blobs" without the "for"

Reply:
we should probably pretty format the namespaces as well

Reply:
Removed "for"

@renaynay what changes to the namespaces would you like to see? The current format is the following:
```
2023-09-28T10:14:51.437+0300	DEBUG	blob	blob/service.go:124	getting blobs	{"height": 118249, "namespaces": ["AAAAAAAAAAAAAAAAAAAAAAAAAEJpDCBNOWAP3dM="]}
2023-09-28T10:14:51.472+0300	DEBUG	blob	blob/service.go:136	receiving blobs	{"height": 118249, "total": 1}
```